vtzero/__init__.py

```diff
@@ -1,5 +1,3 @@
 """vtzero: a python wrapper of https://github.com/mapbox/vtzero."""
 
-import pkg_resources
-
-version = pkg_resources.get_distribution(__package__).version
+version = "0.0.1b2"
```

Comment:
I'd not duplicate the version. We both now that at some point in the history those values will diverge ;)

If we really think using pkg_resource will impact perfs, I'd keep version in the `__init__.py` and then import it from the `setup.py`. Or in a dedicated `version.py` like.

Thoughts ?

Reply:
The performance of using `pkg_resource` is really bad. we removed it from all our package and are now using bump2version to manage updating each code automatically when creating a new tag. I didn't want to add such a thing in this PR because it needs more discussion. 

ref: https://github.com/cogeotiff/rio-tiler/pull/286 https://github.com/pypa/setuptools/issues/510

Reply:
we could totally create a version.py and use something like https://github.com/stac-utils/pystac/blob/main/setup.cfg#L1-L2

Reply:
That's simple and effective and DRY, perfect :)